scripts/opensetup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Install mosquitto
+sudo apt-add-repository ppa:mosquitto-dev/mosquitto-ppa
+sudo apt-get install mosquitto
+sudo apt-get install mosquitto-clients
+ 
+# Generate CA certificate and key
+openssl req -new -x509 -days 365 -extensions v3_ca -keyout my-ca.key -out my-ca.crt 
```

Comment:
https://www.openssl.org/docs/man3.0/man5/config.html
Check openssl config file in Linux. If the input to openssl can be provided using a config file, then that would be preferable